pkg/backend/httpstate/client/client.go

```diff
@@ -208,25 +208,27 @@ type ListStacksFilter struct {
 
 // ListStacks lists all stacks the current user has access to, optionally filtered by project.
 func (pc *Client) ListStacks(
-	ctx context.Context, filter ListStacksFilter) ([]apitype.StackSummary, error) {
+	ctx context.Context, filter ListStacksFilter, inContToken *string) ([]apitype.StackSummary, *string, error) {
 	queryFilter := struct {
-		Project      *string `url:"project,omitempty"`
-		Organization *string `url:"organization,omitempty"`
-		TagName      *string `url:"tagName,omitempty"`
-		TagValue     *string `url:"tagValue,omitempty"`
+		Project           *string `url:"project,omitempty"`
+		Organization      *string `url:"organization,omitempty"`
+		TagName           *string `url:"tagName,omitempty"`
+		TagValue          *string `url:"tagValue,omitempty"`
+		ContinuationToken *string `url:"continuationToken,omitempty"`
 	}{
-		Project:      filter.Project,
-		Organization: filter.Organization,
-		TagName:      filter.TagName,
-		TagValue:     filter.TagValue,
+		Project:           filter.Project,
+		Organization:      filter.Organization,
+		TagName:           filter.TagName,
+		TagValue:          filter.TagValue,
+		ContinuationToken: inContToken,
 	}
 
 	var resp apitype.ListStacksResponse
 	if err := pc.restCall(ctx, "GET", "/api/user/stacks", queryFilter, nil, &resp); err != nil {
-		return nil, err
+		return nil, nil, err
 	}
 
-	return resp.Stacks, nil
+	return resp.Stacks, resp.ContinuationToken, nil
```

Comment:
The `ContinuationToken` field was added earlier, in https://github.com/pulumi/pulumi/pull/7510.